images/installer-origin-release/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# This Dockerfile is used to create the image quay.io/coreos/origin-release:bazel used in CI
+# to build smoke tests.
+# EX: docker build --rm -t this-image:latest .
+
+FROM docker.io/openshift/origin-release:golang-1.10
+
+RUN wget -O /etc/yum.repos.d/vbatts-bazel-epel-7.repo https://copr.fedorainfracloud.org/coprs/vbatts/bazel/repo/epel-7/vbatts-bazel-epel-7.repo
```

Comment:
I'm fine with this, but [with `-P`][1] you don't have to repeat the filename.

[1]: https://github.com/openshift/installer/pull/223#discussion_r216013284

Reply:
ah, good, I'll edit that then